festim/boundary_conditions/fluxes/dissociation_flux.py

```diff
@@ -0,0 +1,32 @@
+from festim import FluxBC, k_B
+import fenics as f
+import sympy as sp
+
+
+class DissociationFlux(FluxBC):
+    """
+    FluxBC subclass for hydrogen dissociation flux.
+    -D(T) * grad(c) * n = -Kd(T) * P
+
+    Args:
+        Kd_0 (float or sp.Expr): dissociation coefficient pre-exponential
+            factor (m^(2)/s)
+        E_Kd (float or sp.Expr): dissociation coefficient activation
+            energy (eV)
+        P (float): partial pressure of H (Pa)
```

Comment:
It is way more likely that users will want to have a time dependent pressure than a time dependent `Kd_0` don't you think?

`P` should also accept a `sp.Expr` object.